### DISCUSSION TITLE
Replit无法部署AList

### DISCUSSION THREAD

**Discussion by xx6z8v** [AlistGo/alist#3587] [Q&A]
如题,无法在Replit部署AList,据网上了解是官方封了,有什么好的解决办法吗

**Comment by ethanjtch** [comment]
https://t.me/sxbai/1962

有大神重新封装了一版，指令要把 alist 改为 replit，其他步骤一致。 

补充说明，需要把 .replit 文件里的 run = "cowsay Configure me!" 改成 run = "chmod +x replit && ./replit server"

**Reply by xx6z8v** [reply]
### 谢谢,我已经解决过了,并有更好的办法
***
1.到Github下载alist包体并重命名为"a.tar.gz"
2.Replit建立任意项目并手动上传"a.tar.gz"
3.项目的shell依次输入"tar zxvf a.tar.gz"和"mv alist ./a"这两条命令
4.修改".replit"文件里的内容为(run = "./a server")
5.可选择删除"a.tar.gz"还有"*.md"等残留文件,或者直接启动项目
***
***后记:我的AList项目已经稳定运行3个月了,没有挂,但很奇怪的是,Replit官方似乎对我的账号做出了限制,现在我的所有项目都无法保存新内容,对我的影响包括但不限于-我的新项目无法继续使用Uptime监控,我的AList无法保存新的储存策略和更新样式,我的阅读项目无法更新书籍进度.请问您是否有遇到过同样问题?***

**Comment by ethanjtch** [comment]
的确简单很多了！

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: xx6z8v ***@***.***>
发送时间: Thursday, June 1, 2023 10:51:31 PM
收件人: alist-org/alist ***@***.***>
抄送: Ethan Chen ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/alist] Replit无法部署AList (Discussion #3587)


谢谢,我已经解决过了,并有更好的办法
________________________________

1.到Github下载alist包体并重命名为"a.tar.gz"
2.Replit建立任意项目并手动上传"a.tar.gz"
3.项目的shell依次输入"tar zxvf a.tar.gz"和"mv alist ./a"这两条命令
4.修改".replit"文件里的内容为(run = "./a server")
5.可选择删除"a.tar.gz"还有"*.md"等残留文件,或者直接启动项目

________________________________

***后记:我的AList项目已经稳定运行3个月了,没有挂,但很奇怪的是,Replit官方似乎对我的账号做出了限制,现在我的所有项目都无法保存新内容,对我的影响包括但不限于-我的新项目无法继续使用Uptime监控,我的AList无法保存新的储存策略和更新样式,我的阅读项目无法更新书籍进度.请问您是否有遇到过同样问题? ***

―
